File: app/src/main/java/com/gasparaiciukas/owntrainer/network/Parsed.java
package com.gasparaiciukas.owntrainer.network;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Parsed {

    @SerializedName("quantity")
    @Expose
    private int quantity;
    @SerializedName("measure")
    @Expose
    private String measure;
    @SerializedName("food")
    @Expose
    private String food;
    @SerializedName("foodId")
    @Expose
    private String foodId;
    @SerializedName("weight")
    @Expose
    private int weight;
    @SerializedName("retainedWeight")
    @Expose
    private int retainedWeight;
    @SerializedName("measureURI")
    @Expose
    private String measureURI;
    @SerializedName("status")
    @Expose
    private String status;

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public String getMeasure() {
        return measure;
    }

    public void setMeasure(String measure) {
        this.measure = measure;
    }

    public String getFood() {
        return food;
    }

    public void setFood(String food) {
        this.food = food;
    }

    public String getFoodId() {
        return foodId;
    }

    public void setFoodId(String foodId) {
        this.foodId = foodId;
    }

    public int getWeight() {
        return weight;
    }

    public void setWeight(int weight) {
        this.weight = weight;
    }

    public int getRetainedWeight() {
        return retainedWeight;
    }

    public void setRetainedWeight(int retainedWeight) {
        this.retainedWeight = retainedWeight;
    }

    public String getMeasureURI() {
        return measureURI;
    }

    public void setMeasureURI(String measureURI) {
        this.measureURI = measureURI;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

}

File: app/src/main/java/com/gasparaiciukas/owntrainer/network/Parsed.kt
package com.gasparaiciukas.owntrainer.network

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Parsed
(
@SerializedName("quantity") @Expose var quantity: Int = 0,
@SerializedName("measure") @Expose var measure: String? = null,
@SerializedName("food") @Expose var food: String? = null,
@SerializedName("foodId") @Expose var foodId: String? = null,
@SerializedName("weight") @Expose var weight: Int = 0,
@SerializedName("retainedWeight") @Expose var retainedWeight: Int = 0,
@SerializedName("measureURI") @Expose var measureURI: String? = null,
@SerializedName("status") @Expose var status: String? = null
)